### PR TITLE
Add new word and style rules #3795

### LINKED ISSUE #3795: Style Guide update: add words and phrases, forbid Latin, rule on measurement unit spacing
**Style Guide >> Voice and language**
Discourage the use of Latin along with jargon and idioms. It's true in the Google Style guide and it's a typical and uncontroversial style guide rule.

**Style Guide >> Word & Phrase List**
__minikube__ over __Minikube__
__hostname__ over __host name__

**Style Guide >> Formatting standard and conventions**
Copy the salient Google Style rules for spacing in units of measurement (don't bother with angle degrees and money).

### PR DESCRIPTION
**Pull request by RichardJJG**
Fixes #3795

### CODE REVIEW

File: docs/help/contributor/style-guide/style-and-formatting.md [hunk 1]
```diff
@@ -28,11 +28,24 @@ word of the heading, except for proper nouns or acronyms.
...
+|Using feature      | Using Feature
 |Using HTTPS         | Using https
 
 ## Create descriptive links
 
 |:white_check_mark: Correct                                     |:no_entry: Incorrect
 |---------------------------------------|------
 |Check out [this excellent article](https://medium.com/@heyoka/Correctnt-use-click-here-f32f445d1021) explaining <br> what makes a good hyperlink    | Check out the article [here](https://medium.com/@heyoka/Correctnt-use-click-here-f32f445d1021)
+
+## Spaces in units of measurement
```
Comment (snneji) [inline docs/help/contributor/style-guide/style-and-formatting.md:40]:
```suggestion
## Use spaces in units of measurement
```

Comment (snneji) [reply docs/help/contributor/style-guide/style-and-formatting.md:40]: Start with a verb to be consistent with other headings on the page

**Review by snneji — CHANGES_REQUESTED** [review]
One small tweak to the heading. Otherwise looks good

**Review by abrennan89 — APPROVED** [review]
/lgtm

**Pull request merged**